app/controllers/review_mapping_controller.rb

```diff
@@ -325,82 +331,103 @@ def delete_metareview
   def list_mappings
     flash[:error] = params[:msg] if params[:msg]
     @assignment = Assignment.find(params[:id])
-    # ACS Removed the if condition(and corresponding else) which differentiate assignments as team and individual assignments
+    # ACS Removed the if condition(and corressponding else) which differentiate assignments as team and individual assignments
     # to treat all assignments as team assignments
     @items = AssignmentTeam.where(parent_id: @assignment.id)
     @items.sort_by(&:name)
   end
 
+  def create_team(participants,assignment_id, teams)
```

Comment:
Can you break down this method into smaller methods? Please add some comments over what this method does